src/predicate.ts

```diff
@@ -117,7 +118,7 @@ export interface FieldRangePredicate extends FieldPredicateBase {
    * @maxItems 2
    * @minItems 2
    */
-  range: (number | DateTime | null)[];
+  range: (number | DateTime | null | SignalRef)[];
```

Comment:
Should this be `(number | DateTime | null | SignalRef)[]` or `(number | DateTime | null)[] | SignalRef`?

Reply:
Ah yes, technically both.